### PR TITLE
Adjust documentation

### PR DESCRIPTION
**Pull request by mrproliu**
Fix https://github.com/apache/skywalking/issues/11988

### CODE REVIEW

File: docs/en/setup/configuration/traffic.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Access Log Module
+# Traffic
 
-The access log module is used to collecting the network access logs from the [Kerbernetes Module](process_discovery/kubernetes.md),
+The traffic is used to collecting the network access logs from the [Service Discovery](service-discovery.md),
```
Comment (wu-sheng) [inline docs/en/setup/configuration/traffic.md:3]:
```suggestion
The traffic is used to collecting the network access logs from services through the [Service Discovery](service-discovery.md),
```

File: docs/en/setup/overview.md [hunk 2]
```diff
@@ -33,24 +33,17 @@ You can quickly build your Rover according to the following examples:
...
-3. [Process Discovery Module](configuration/process-discovery.md).
-4. [Profiling Module](./configuration/profiling.md).
-5. [Access Log Module](./configuration/accesslog.md).
+1. [Common](./configuration/common.md).
+2. [Service Discovery](configuration/service-discovery.md).
+3. [Traffic](./configuration/traffic.md).
+4. [Profiling](./configuration/profiling.md).
 
-Also, You could using [Overriding Setting](./configuration/override-settings.md) feature to setup the configuration.
+Also, You could use [Overriding Setting](./configuration/override-settings.md) feature to setup the configuration.
```
Comment (wu-sheng) [inline docs/en/setup/overview.md:46]:
```suggestion
Also, You could use [Overriding Setting](./configuration/override-settings.md) feature to adjust the configurations.
```

File: docs/menu.yml [hunk 3]
```diff
@@ -23,34 +23,24 @@ catalog:
...
-            - name: Access Log Module
-              path: /en/setup/configuration/accesslog
         - name: Examples
           catalog:
-            - name: Deploy
-              catalog:
-                - name: Deploy on Kubernetes
-                  path: /en/setup/examples/deploy/kubernetes/readme
+            - name: Deploy on K8s
+              path: /en/setup/examples/deploy/kubernetes/readme
```
Comment (wu-sheng) [inline docs/menu.yml:43]: You could set it as Deployment directly to replace the whole examples menu, there is no other examples, and this is the official deployment I think, rather han an example.

Please remove the examples folder too.

Comment (mrproliu) [reply docs/menu.yml]: Updated, please help to re-check

**Pull request merged**